packages/suite/src/actions/wallet/sendFormSpecific/bitcoinActions.ts

```diff
@@ -4,7 +4,7 @@ import { Output } from '@wallet-types/sendForm';
 import { networkAmountToSatoshi } from '@wallet-utils/accountUtils';
 import { getLocalCurrency } from '@wallet-utils/settingsUtils';
 import { SEND } from '@wallet-actions/constants';
-import { INPUT_SEQUENCE } from '@wallet-constants/sendForm';
+import { BITCOIN_INPUT_SEQUENCE } from '@wallet-constants/sendForm';
```

Comment:
all others constants prefix are "BTC", is there any reason why this one has "BITCOIN" ?

Reply:
maybe it should be called "BTC_RBF_SEQUENCE" ?

Reply:
no :)